### ISSUE TITLE
[DOC] Installation instruction incomplete

### ISSUE THREAD

**Issue by b1llab0ng** [appneta/tcpreplay#568]
**Describe the bug**
Installation instructions in _README.md_ file and on the [Download and installation](http://tcpreplay.appneta.com/wiki/installation.html ) web page are outdated.  Missing coverage for _autogen.sh_ script.

**To Reproduce**
1. GIT Clone fresh Master source in a new empty folder
2. Attempt to execute _./configure_ per installation instructions fails, file not found (without prior executing _autogen.sh_ script, as not mentioned in instructions, the _configure_ file does not exist yet).

**Expected behavior**
Installation instructions expected to be complete from fresh GIT cloned copy.
Installation instructions to be updated to add prior to execute ./configure, the step to execute the _autogen.sh_ script in order to get the necessary _configure_ script file.

**Screenshots**
See * mark below:
![Missing autogen coverage](https://user-images.githubusercontent.com/55287503/67045570-90023780-f0fc-11e9-92aa-8886e845a89e.png)

**System (please complete the following information):**
 - OS: Debian
 - OS version: 4.9.189-3 (2019-09-02) x86_64 GNU/Linux
 - Tcpreplay Version 4.3.2 (build git:v4.3.2-2-ga00fd476)

**Additional context**
N/A

**Comment by fklassen** [comment]
You are missing [the Donwloads](https://github.com/appneta/tcpreplay#downloads) step in the doc. Release tarballs have `./configure` script.

The `configure` script is not included in the source code, as is common practise. Proper generation of the script requires `autogen` which is very version dependent. Developers create releases with `sudo make test && make dist`. Regular users use releases.

**Comment by b1llab0ng** [comment]
Ok.  I further read the instructions and figured it out. It's true that the tar ball contains the _configure_ file.
  I intermingled the download instructions for developer cloning from GIT and followed simple installation instructions for std user, leading to why I opened this ticket at first.  
  Although, perhaps it would be a good addition for complementing instructions, to provide the installation instructions for when cloning from GIT; then prior running the _autogen.sh_ script is necessary for obtaining the _./configure_.

**Comment by fklassen** [comment]
Point taken. Will target next fix release.

**Comment by resolutedreamer** [comment]
Hello. I think this is still a problem. I downloaded the source code from the version linked in the Download page and that zip file did not contain _configure_ file

**Comment by fklassen** [comment]
Just download the release https://github.com/appneta/tcpreplay/releases/tag/v4.5.1

**Comment by resolutedreamer** [comment]
Yes sir, I went to the exact same URL that you provided, and I clicked on the "Source code (zip)" button.

![image](https://github.com/user-attachments/assets/09c99207-9aca-48ca-b4a0-f3390cdf3b87)

And then after opening the downloaded file, the contents of the zip appeared as follows 

![image](https://github.com/user-attachments/assets/6127139a-eff4-4d99-8ddf-a89d7f7606f7)

Which does not contain a configure file.

**Comment by fklassen** [comment]
Fixed in #1009. README's 'Simple directions for Unix users' only ever showed ./configure && make && sudo make install, which only works from a release tarball (configure ships pre-generated there via make dist). A fresh git clone has no configure script until autogen.sh generates it - confirmed the gap was still there on current v4.5.3-beta1, 6 years after this was first reported.

Split the section into release-tarball vs. git-checkout cases, with autogen.sh (and its prereqs) called out for the latter. Left docs/INSTALL alone since that's the stock GNU autotools boilerplate template, not project-specific - the README was where this issue and a typical reader would actually look. The wiki installation page mentioned in the original report is external and outside what a repo PR can fix.
